README.md

```diff
@@ -112,6 +112,12 @@ Unit test name in this case will be "AddTwoNumbers :: Add two numbers", which is
 
 ![Screenshot of scenario test run](scenario-test-run-screenshot.png)
 
+#### Resharper
+
+You can use Resharper test runner to execute the tests but it has it's limitations. These limitations are not limited to Xunit.Gherkin.Quick but also apply to some core Xunit features.
+
+For Xunit.Gherkin.Quick the limitations are around Scenario Background usage. This is easily noticed when you run all tests with the Visual Studio harness and compare the output to Resharper's test harness. Resharper appears to have run fewer tests - in fact it will only run a single test per Scenario, regardless of how many Examples are defined.
```

Comment:
It says "Scenaio" but should say "Scenario Outline", because examples are defined for Scenario Outline, not Scenario.
Also, let's move this text (either both paragraphs or just this one) to the Scenario Outlines documentation - https://github.com/ttutisani/Xunit.Gherkin.Quick/blob/master/docs/scenario-outline.md

That's where people will expect this hint and will be more relevant.